.vscode/settings.json

```diff
@@ -1,35 +1,17 @@
 {
-  "[typescriptreact]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode",
-    "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": "explicit"
-    }
-  },
-  "[typescript]": {
-    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
-    "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": "explicit"
-    }
-  },
-  "[javascript]": {
-    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
-    "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": "explicit"
-    }
-  },
   "editor.detectIndentation": false,
   "editor.tabSize": 2,
   "jestrunner.jestCommand": "npx dotenv -e .env.test.local -- node --experimental-vm-modules 'node_modules/.bin/jest'",
-  //// "jestrunner.jestCommand": "dotenv -e .env.test.local -- nox  --experimental-vm-modules 'node_modules/.bin/jest'",
   "jestrunner.configPath": "jest.config.js",
   "typescript.tsdk": "node_modules/typescript/lib",
-  "[prisma]": {
-    "editor.defaultFormatter": "Prisma.prisma"
-  },
   "[javascript][typescript][typescriptreact]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
     "editor.codeActionsOnSave": {
       "source.fixAll.eslint": "explicit",
-      "source.organizeImports": "always"
+      "source.organizeImports": "never"
```

Comment:
this rule was clearing out imports before i got a chance to use them. I usually type out the import, hit "save" so that it auto-sorts it in the list, and then start using it. But that doesn't work when the IDE just deletes the unused import